src/com/palmergames/bukkit/towny/TownyFormatter.java

```diff
@@ -141,8 +142,10 @@ public static String getTime() {
 	}
 
 	/**
-	 * 
-	 * @param resident
+	 *  Gets the metadata of a Resident

```

Comment:
instead of metadata maybe say status screen

Reply:
Resolved in c27c7e04891a59b2778dd383f2b573495894c26d